strix/utilities/utils.py

```diff
@@ -80,31 +80,39 @@ def parse_datalist(filelist, format="auto", has_unlabel: bool = False):
         label: str = DatalistKeywords.LABEL.value
         unlabel: str = DatalistKeywords.UNLABEL.value
         if label not in datalist:
-            raise GenericException(f"Your datalist does not contain '{label}' key.")
-        elif "tensorImageSize" in datalist:
-            training_list = datalist["training"]
-            num = len(training_list)
-            dataset = []
-            for i in range(num):
-                case_data = {}
-                if "image" in training_list[i]:
-                    image_relpath = training_list[i]["image"]
-                    case_data['image'] = os.path.abspath(image_relpath)
-                if "label" in training_list[i]:
-                    label_relpath = training_list[i]["label"]
-                    case_data['label'] = os.path.abspath(label_relpath)
-                if "mask" in training_list[i]:
-                    mask_relpath = training_list[i]["mask"]
-                    case_data['mask'] = os.path.abspath(mask_relpath)
-                dataset.append(case_data)
-            return dataset
+            if "tensorImageSize" in datalist:
+                training_list = datalist["training"]
+                num = len(training_list)
+                data_path =Path(filelist).parent
+                dataset = []            
+                if not os.path.exists(str(data_path/training_list[0]["image"])):
```

Comment:
Since data_path is already a `Path`, change `os.path.exists(str(data_path/training_list[0]["image"]))` to `(data_path/training_list[0]["image"]).exists()`